modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -37,6 +37,37 @@ ifdef::ovn[]
 - A protocol that is one of the following protocols: TCP, UDP, and SCTP
 endif::ovn[]
 
+[IMPORTANT]
+====
+If your egress firewall includes a deny rule for `0.0.0.0/0` then access to your {product-title} API servers is blocked.
+To ensure pods can continue to access the {product-title} API servers, you must include the IP address range that the API servers listen on in your egress firewall rules, as in the following example:
+
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+  namespace: <namespace> <1>
+spec:
+  egress:
+  - deny:
+      cidrSelector: 0.0.0.0/0 <2>
+    type: Deny
+  - to:
+      cidrSelector: <api_server_address_range> <3>
+    type: Allow
```

Comment:
The allow rules must be before Deny 0.0.0./0, or else it won't take effect for this allow rule.